nntrainer/include/layers.h

```diff
@@ -177,7 +140,7 @@ class Layer {
    * @brief     Optimizer Setter
    * @param[in] opt Optimizer
    */
-  virtual void setOptimizer(Optimizer opt) = 0;
+  void setOptimizer(Optimizer opt);
```

Comment:
Add virtual?

What do you want?
```
Layer * a = new InputLayer();
a->setOptimizer();
```
Do you want to access ```Layer::setOptimizer()``` (current code)? or ```InputLayer::setOptimizer()``` (add virtual)?

Reply:
virtual added. Thanks.